src/sentry/static/sentry/app/components/placeholder.tsx

```diff
@@ -4,33 +4,45 @@ import PropTypes from 'prop-types';
 
 import space from 'app/styles/space';
 
-const Placeholder = styled(props => {
+const defaultProps = {
+  shape: 'rect' as 'rect' | 'circle',
```

Comment:
why do we need `as 'rect'` here?

*edit* I see it's trying to cast it as `'rect'|'circle'` (and that `defaultProps` is not a type)

Reply:
Yeah without the `as` statement typescript would infer the type as string, when I want it to be just rect or circle.